### ISSUE TITLE
(aws-elasticloadbalancingv2-targets): `stickiness.enabled` not supported for `HTTPS` target group type

### ISSUE THREAD

**Issue by jaecktec** [aws/aws-cdk#17774]
### What is the problem?

When creating an `ApplicationTargetGroup` for a `LambdaTarget` by using the `new ApplicationTargetGroup({...})` constructor, 
`stickiness.enabled` is by default specified (true/false), but is not supported for a 'HTTPS' target group type

### Reproduction Steps

```ts
const applicationTargetGroup = new ApplicationTargetGroup(this, 'tg', {
  vpc,
  targetGroupName: `ECS-TG`,
  targets: [new LambdaTarget(func)],
});

httpsListener.addTargetGroups('target', {
  conditions: [ListenerCondition.pathPatterns([
    ...
  ])],
  priority: ...
  targetGroups: [applicationTargetGroup],
});
```

### What did you expect to happen?

Should create a TargetGroup that works with LambdaTargets

### What actually happened?

Creates a TargetGroup with 

```ts
stickiness.enabled = "true"
```

### CDK CLI Version

2.0.0-rc.33 (build 336ff5e)

### Node.js Version

v14.16.1

### OS

macOS Big Sur 11.4

### Language

Typescript

### Language Version

3.9.10

### Other information

It can be worked-around by manually deleting this option:

```
applicationTargetGroup.setAttribute('stickiness.enabled', undefined)
```

**Comment by ilko-rbi** [comment]
same here with v1.134.0 - [here](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#target-group-attributes) the link to the documentation for the supported attributes on a Target Group

**Comment by njlynch** [comment]
Related to https://github.com/aws/aws-cdk/pull/17111 and https://github.com/aws/aws-cdk/pull/17271.

**Comment by njlynch** [comment]
Ah, at closer inspection, this is identical to what was fixed by #17271. This fix has not yet been ported up to v2, but should go out with the v2 release next week.
